helm-go-lib/main.go

```diff
@@ -174,6 +176,63 @@ func uninstallRelease(namespace string, kubeToken string, apiServer string, caFi
 	return C.CString("ok")
 }
 
+//export upgradeChart
+func upgradeChart(namespace string, kubeToken string, apiServer string, caFile string, releaseName string, chartName string, overrideValues string) *C.char {
```

Comment:
Should there be a `version string` param to this function (similar to `installChart`)? Which would specify the version we're upgrading _to_..

Reply:
Indeed, the very point of why we would want to upgrade in the first place 🙈